ichnaea/service/locate.py

```diff
@@ -97,14 +107,14 @@ def map_data(data, client_addr=None):
     return mapped
 
 
-class AbstractResult(object):
-    """The result of a location provider query."""
+class AbstractLocation(object):
+
```

Comment:
What pep8 rule is this? My pep8/flake8 doesn't warn me about this one. My flake8 config has "ignore = E226,N802,N806,N811", with the pep8-naming plugin installed.